README.md

```diff
@@ -208,7 +197,7 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@simple-kmod"]
+              command: ["/bin/sh", "-c", "modprobe -r simple-kmod"]
         securityContext:
           privileged: true
```

Comment:
This might be enough?
```suggestion
          capabilities:
            add: [SYS_MODULE]
```

Reply:
Yes. I'm pretty sure it's enough.

Reply:
This issue need further investigation as SELinux is also involved. New task opened:
[https://issues.redhat.com/browse/MGMT-10213](url)

Reply:
Working link - https://issues.redhat.com/browse/MGMT-10213